src/components/ContinueButton.js

```diff
@@ -13,7 +13,8 @@ const ContinueButton = (props) => (
     disabled={props.disabled}
     id="continue-button"
     type="submit"
-    icon={props.type === "begin" ? <ButtonArrow /> : null}
```

Comment:
I don't think we want to remove the icon, we just need to prevent it from being read out by screen readers by adding the aria-hidden and focusable tags